src/vm/analysis/type_checker/natives/sequences.rs

```diff
@@ -312,7 +313,7 @@ pub fn check_special_len(
     check_argument_count(1, args)?;
 
     let collection_type = checker.type_check(&args[0], context)?;
-    runtime_cost!(cost_functions::ANALYSIS_ITERABLE_FUNC, checker, 1)?;
+    runtime_cost(ClarityCostFunction::AnalysisIterableFunc, checker, 0)?;
```

Comment:
This cost function doesn't take any input, so it doesn't matter what this number is. Not sure why it used to be 1, I think it was a typo. We are using 0 to represent a lack of input. The reason we are using this "ignored" parameter instead of an Optional is that it makes it easier to vote in new cost functions that will take args.